src/main/java/ch/wisv/areafiftylan/teams/service/TeamServiceImpl.java

```diff
@@ -113,6 +112,10 @@ public Team update(Long teamId, TeamDTO input) {
 
     @Override
     public Team delete(Long teamId) {
+        List<TeamInviteResponse> invites = findTeamInvitesByTeamId(teamId);
```

Comment:
I don't know if cascading could work, since the `TeamInviteToken `has the `Team`, so when the `Team` gets deleted it has no knowledge of `TeamInviteToken`s.